HACKING/README.md

```diff
@@ -0,0 +1,19 @@
+# Contributing to this project
+
+First of all, thanks! Second of all, this is pretty much just a stub right now, but here's how you use the Dockerfile.
+
+## Dockerfile
+ Run with:
+ ```
+podman build . --tag esp-idf
```

Comment:
Might be good to link to podman install docs?

Reply:
I feel like that's out of scope, but It's no trouble to add them.

Reply:
imo if you're inviting people to help, it's worth giving more guidance, especially as podman is not as broadly known as other tools.